cmd/config/internal/commands/cmdset_test.go

```diff
@@ -751,8 +751,8 @@ spec:
   list: # {"$ref":"#/definitions/io.k8s.cli.setters.list"}
   - 0
  `,
-			errMsg: `list in body must be of type integer: "string"
-list in body must be of type integer: "boolean"
+			errMsg: `list[1] in body must be of type integer: "string"
```

Comment:
These error messages are more specific, so that's good, but what made this change?

Reply:
I think this is because of the kube-openapi upgrade, though I can't find the PR in kube-openapi that made this change.